### ISSUE TITLE
Available fine-tuned checkpoints for inference?

### ISSUE THREAD

**Issue by yungsinatra0** [LUMIA-Group/FourierTransformer#2]
Hello, are there any fine-tuned checkpoints (for example on CNNDM dataset) available for inference? I don't have access to a machine that I can use to pre-train & fine-tune the model at the moment, but I'd like to try this model out for summarization. Thanks!

**Comment by ZiweiHe** [comment]
Hi,

Thank you for your interests, I've uploaded the checkpoint ready for inference on CNN/DM to [huggingface] (https://huggingface.co/ziweihe/FourierTransformer-cnndm). Please check it out there.  The model finetuned on ELI5 will be uploaded in a day or two.

Please leave a comment if you have further questions.

Ziwei

**Comment by yungsinatra0** [comment]
@ZiweiHe Thank you very much for uploading the checkpoint! I will give it a try and let you know if I have any issues :)
